scripts/qa_crowbarsetup.sh

```diff
@@ -3232,53 +3232,68 @@ function oncontroller_manila_generic_driver_setup()
     local sec_group="manila-service"
     local neutron_net=$sec_group
 
-    local ret=$(wget --progress=dot:mega -nc -O $service_image_name "$service_image_url" 2>&1 >/dev/null)
-    if [[ $ret =~ "already there; not retrieving" ]]; then
+    local ret=$(wget -nc -O $service_image_name "$service_image_url" 2>&1 >/dev/null)
+    if [[ $ret =~ "already there; not retrieving" ]] || [[ $ret =~ "200 OK" ]]; then
```

Comment:
can you provide the concrete example? 

man wget

```
...
       -N
       --timestamping
           Turn on time-stamping.
...
```
is not really clear to me